packages/manager/modules/pci/src/projects/project/private-networks/add/add.controller.js

```diff
@@ -0,0 +1,194 @@
+import get from 'lodash/get';
+import partition from 'lodash/partition';
+import set from 'lodash/set';
+import sortBy from 'lodash/sortBy';
+import zipWith from 'lodash/zipWith';
+
+import {
+  DEFAULT_VLAN_ID, DEFAULT_CIDR, VLAN_ID, VLAN_GUIDES,
+} from './add.constants';
+import IP from './ip';
+
+const CONTAINER_NAME = 'pci.projects.project.privateNetwork.add';
+
+export default class NetworkAddCtrl {
+  /* @ngInject */
+  constructor(
+    $translate,
+    coreConfig,
+    CucCloudMessage,
+    CucRegionService,
+    PciPrivateNetworksList,
+    PciPrivateNetworksAdd,
+  ) {
+    this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
+    this.CucRegionService = CucRegionService;
+    this.PciPrivateNetworksList = PciPrivateNetworksList;
+    this.PciPrivateNetworksAdd = PciPrivateNetworksAdd;
+
+    this.USER_REGION = coreConfig.getRegion();
```

Comment:
There is no benefit to refer to the a class reference since `USER_REGION` is only used in `getApiGuide` method.

I think you can move the assignation directly inside the `getApiGuide` method.